v2/internal/pkg/cli/executor.go

```diff
@@ -1026,7 +1028,21 @@ func (o *ExecutorSchema) CollectAll(ctx context.Context) (v2alpha1.CollectorSche
 		o.closeAll()
 		return v2alpha1.CollectorSchema{}, err
 	}
+
+	// CLID-230 rebuild-catalogs
 	oImgs := oCollector.AllImages
+	if o.alphaCtlgFilter && (o.Opts.IsMirrorToDisk() || o.Opts.IsMirrorToMirror()) {
+		results, delImgs, err := o.ImageBuilder.RebuildCatalogs(ctx, oCollector)
+		if err != nil {
+			o.closeAll()
+			return v2alpha1.CollectorSchema{}, err
+		}
+		oImgs = excludeImages(oImgs, delImgs)
```

Comment:
It's a good thing to exclude the images that failed to rebuild from the mirroring in order not to stop the mirroring. 
Nevertheless, these images should be added to the error file we generate, so that we make sure the user is aware that some catalogs are missing.
Also when a catalog failed to rebuild, do we exclude the operator images that are related to that catalog?